packages/picasso/src/FileList/FileList.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { forwardRef, HTMLAttributes } from 'react'
+import cx from 'classnames'
+import { ProgressBar } from '@toptal/picasso-lab'
+import { BaseProps } from '@toptal/picasso-shared'
+import { makeStyles, Theme } from '@material-ui/core/styles'
+
+import Container from '../Container'
+import Typography from '../Typography'
+import Tooltip from '../Tooltip'
+import Button from '../Button'
+import { Link16, Trash16, Close16 } from '../Icon'
+import styles from './styles'
+
+interface File {
+  uploading: boolean
+  progress?: number
+  error?: string
+  name: string
+}
+
+interface FileListItemProps {
+  file: File
+  onRemove?: (fileName: string) => void
+}
+
+export interface Props extends BaseProps, HTMLAttributes<HTMLDivElement> {
+  files: File[]
+  onItemRemove?: (fileName: string) => void
+}
+
+const useStyles = makeStyles<Theme>(styles, {
+  name: 'FileList'
+})
+
+const FileListItem = ({ file, onRemove }: FileListItemProps) => {
+  const { uploading, progress = 0, error, name } = file
+
+  const classes = useStyles()
+
+  const renderUploading = (
```

Comment:
It's not a function, so it shouldn't contain a verb in its name.

```suggestion
  const renderUploading = () => (
```

or 

```suggestion
  const uploadingNode = (
```